velox/common/base/Counters.h

```diff
@@ -128,4 +119,28 @@ constexpr folly::StringPiece kMetricSpillWriteTimeMs{
 
 constexpr folly::StringPiece kMetricFileWriterEarlyFlushedRawBytes{
     "velox.file_writer_early_flushed_raw_bytes"};
+
+constexpr folly::StringPiece kMetricArbitratorNumRequests{
```

Comment:
Can you update document for the new ones? Thanks!

Reply:
counter document is in .cpp per Velox style, which is different from presto